bootstrap.sh

```diff
@@ -0,0 +1,14 @@
+echo "Setting up docker-compose.yml file, if it doesn't exist already"
+make
```

Comment:
Do we want this commit to add and commit a `docker-compose.yml` with this PR?

And, do we wish to use `make -B` to force a build of the docker-compose file?  It isn't clear to me how the make targets will determine whether or not an existing `docker-compose.yml` will be overwritten.

Reply:
OK, so I made bootstrap a make target, so I he developer can choose to do `make -B bootstrap` if they want.

As far as committing docker-compose.yml, I'd like to do a few experiments, so maybe consider that outside the scope for this particular PR

Reply:
OK, I think we can't commit docker-compose.yml, because bind mount paths are absolute.  If we can work around that, it'd work.  But for now, that's a showstopper.  So let's just not commit it for now

We'd have /home/bseeger conflicting with /home/birkland conflicting with /home/emetsger

Reply:
Right. Yes.  I've tried various tricks with `--project-directory` but nothing gets rid of the absolute paths.  Very annoying.